app.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
+
+import * as dotenv from 'dotenv';
+dotenv.config();
```

Comment:
This is a step in the right direction but how do we know which items are available for config in the first place? It's good to have a static config file for development and then use env for production.